packages/core/test/Address/util.test.ts

```diff
@@ -1,19 +1,70 @@
-import { Address } from '../../src';
-
-jest.mock('../../src/CSL/parseCslAddress');
-const { parseCslAddress } = jest.requireMock('../../src/CSL/parseCslAddress');
+import * as parseCslAddress from '../../src/CSL/parseCslAddress';
+import { Address, Cardano } from '../../src';
+import { CSL as SerializationLib } from '../../src/CSL';
 
 describe('Address', () => {
+  const parseCslAddressSpy = jest.spyOn(parseCslAddress, 'parseCslAddress');
```

Comment:
I had to modify the way it's mocked so I could use the real implementation in the new tests